### PR TITLE
Tided up and added features to scripts.js

### PR DESCRIPTION
**Pull request by NightfallAlicorn**
Features:
- . and t now shows a ± bot symbol before the user’s name to know exactly no one else saw it. More test characters can be added into an array now instead of coders needing to make a new or-condition.
Example: (20:47:13) ±Nightfall Alicorn: t
- Logon message “You are banned!” replaced with “You are banned!  Appeals can be made here. Attempted evading will extend punishments.” The “here” text is a link to the disciplinary forum page for those who do not know of it.

Script Cleanup:
- The { and } brackets have been tided up.
- Single line codes that had no pointy bracket containers for conditions now added.
- Replaced most mixed double and single quotes areas as double ones.
- Added spaces for coding calculations.
- Command symbol condition easier to handle and readable.
Example: [“/”, “!”].indexOf(message.charAt(0) instead of message[0] == “/”. message[0] could be misread as an array and not as a charAt. More command symbols can be added the array now.
- Replaced /* and */ note comments with // to make it safer for debugging.
- Removed unnecessary empty double lines with 1 empty line.
- Fixed tab spacing errors found.
- require variable wasn’t declared properly. It now is.
- Replaced other message[0] with message.charAt(0) to know it’s a charAt code and not an array.
- Replaced == and != with === and !== for cleaner solider code. Was told it improves performance this way.
- Added comments of end-of-functions to know it’s the end of a function. Handy for knowing the end of large ones.

Hadn’t found any errors after testing, on my server, the changes. Just hadn’t been able to see the new banned logon message but format looks okay on my eval printHtml client script testing.

### CODE REVIEW

File: scripts.js [hunk 1]
```diff
@@ -480,11 +477,14 @@ init : function() {
             var proxy_ip = lines[k].split(":")[0];
             if (proxy_ip !== 0) proxy_ips[proxy_ip] = true;
         }
-    }
+    } // enf of addProxybans
```
Comment (turbedi) [inline scripts.js]: *end

File: scripts.js [hunk 2]
```diff
@@ -1412,7 +1444,7 @@ afterLogIn : function(src) {
     /*if (isAndroid(src)) {
         normalbot.sendMessage(src, "New android version back on Play Store! See: http://pokemon-online.eu/threads/po-android-play-store-revival.29571/");
     }*/
-}, /* end of afterLogin */
+}, // end of afterLogin
```
Comment (turbedi) [inline scripts.js]: // end of afterLogIn

File: scripts.js [hunk 3]
```diff
@@ -1508,8 +1549,6 @@ afterChangeTeam : function(src)
 
 }, /* end of afterChangeTeam */
```
Comment (turbedi) [inline scripts.js]: change this to "//" ?

File: scripts.js [hunk 4]
```diff
@@ -1818,8 +1858,8 @@ beforeChatMessage: function(src, message, chan) {
         return;
     } /* end of commands */
```
Comment (turbedi) [inline scripts.js]: change this to "//" ?

**Comment by NightfallAlicorn** [comment]
Thanks. I just fixed "end" typo on line 480.

Yea, looks like I missed that line.

**Comment by CrystalMoogle** [comment]
no

**Pull request closed without merging**